### PR TITLE
Working compilation on debian

### PR DESCRIPTION
**Pull request by plasticle**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,11 @@ Note this has only been tested on MacOS X. You will need to adjust the
 build script a little bit and probably fix case-sensitive path on Linux
 but I believe it should work with minimal work.
 
+Debian GNU/Linux dependencies:
+- Inkscape
```
Comment (fabiensanglard) [inline README.md]: nit: Would be awesome to provide an "apt-get install" command to get these.

**Comment by fabiensanglard** [comment]
Sorry to be a pain but please squash these two commits into one, the goal is to have one clean commit.

**Comment by plasticle** [comment]
Hey, just wanted to say: Thanks for the books. They are great! (I have the print copies as well :))
The amount of detail that you go into is amazing. Thanks for looking at my PRs!

**Comment by fabiensanglard** [comment]
I just noticed that this patch breaks the build. It moves the "cd src" to after calls to svgToPNG (hence the error message "find: ‘screenshots_svg’: No such file or directory").

I should have reviewed the CL better and also written path resilient bash script so i will fix it.

**Pull request merged**